cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -50,11 +52,17 @@ if(ANDROID)
     hunter_user_error("Could not find android architecture. Please set ANDROID_ARCH_NAME in your toolchain or use CMake 3.7+")
   endif()
   set(configure_command
-      # Ignored. Prevents script from checking host uname
-      RELEASE=2.6.37
-      SYSTEM=android
-      ARCH=${ANDROID_SSL_ARCH}
-      ${configure_command})
+      ${configure_command} ${CMAKE_C_FLAGS})
+
+  # Latest OpenSSL (1.1.1) still does not have correct android targets, so set generic with our CFLAGS
+  string(COMPARE EQUAL "${ANDROID_SSL_ARCH}" "arm64" _is_arm64)
+  string(COMPARE EQUAL "${ANDROID_SSL_ARCH}" "mips64" _is_mips64)
+  string(COMPARE EQUAL "${ANDROID_SSL_ARCH}" "x86_64" _is_x8664)
+  if (_is_arm64 OR _is_mips64 OR _is_x8664)
+    set(configure_opts "linux-generic64")
```

Comment:
So originally I linked to the Configure script and you said you wanted the one with the Android config, so I linked to the old config.
The old method (config) has the Android support but it mangles CFLAGS and assumes GCC.
The new method (Configure) does not specify Android at all because it works the same as Linux. As you can see in the config script, it simply uses linux-generic32 or linux-generic64 along with the required CFLAGS that Hunter already uses. Therefore, there was no point in using the config script.

The RELEASE, SYSTEM and ARCH are specific for the ./config script and not required when using Configure.